tests/trainer/test_trainer.py

```diff
@@ -182,6 +182,17 @@ def _optimizer_step(epoch, batch_idx, optimizer, optimizer_idx,
         # clear gradients
         optimizer.zero_grad()
 
+    model = EvalModelTemplate()
+    schedule = {1: 2, 3: 4}
+
+    trainer = Trainer(
+        accumulate_grad_batches=schedule,
+        limit_train_batches=0.1,
+        limit_val_batches=0.1,
+        max_epochs=2,
+        default_root_dir=tmpdir,
+    )
+
```

Comment:
this redefines the trainer and schedule, and the pytest.mark.parameterize has no effect. 
Is this intentional?

Reply:
no, this was a rebasing issue...